src/components/theme/App/App.jsx

```diff
@@ -259,72 +259,76 @@ export const fetchContent = async ({ store, location }) => {
-          !hasApiExpander('navigation', getBaseUrl(location.pathname))
-        ) {
-          return dispatch(
-            getNavigation(
-              getBaseUrl(location.pathname),
-              config.settings.navDepth,
-            ),
-          );
-        }
+      {
+        key: 'content',
+        promise: ({ location, store }) =>
+          __SERVER__ && fetchContent({ store, location }),
       },
-    },
-    {
-      key: 'types',
-      promise: ({ location, store: { dispatch } }) => {
-        // Do not trigger the types action if the expander is present
-        if (
-          __SERVER__ &&
-          !hasApiExpander('types', getBaseUrl(location.pathname))
-        ) {
-          return dispatch(getTypes(getBaseUrl(location.pathname)));
-        }
+      {
+        key: 'navigation',
+        promise: ({ location, store: { dispatch } }) => {
+          // Do not trigger the navigation action if the expander is present
+          if (
+            __SERVER__ &&
+            !hasApiExpander('navigation', getBaseUrl(location.pathname))
+          ) {
+            return dispatch(
+              getNavigation(
+                getBaseUrl(location.pathname),
+                config.settings.navDepth,
+              ),
+            );
+          }
+        },
       },
-    },
-    {
-      key: 'workflow',
-      promise: ({ location, store: { dispatch } }) =>
-        __SERVER__ && dispatch(getWorkflow(getBaseUrl(location.pathname))),
-    },
-  ]),
-  injectIntl,
-  connect(
-    (state, props) => ({
-      pathname: props.location.pathname,
-      token: state.userSession.token,
-      userId: state.userSession.token
-        ? jwtDecode(state.userSession.token).sub
-        : '',
-      content: state.content.data,
-      apiError: state.apierror.error,
-      connectionRefused: state.apierror.connectionRefused,
-    }),
-    null,
-  ),
-)(App);
+      {
+        key: 'types',
+        promise: ({ location, store: { dispatch } }) => {
+          // Do not trigger the types action if the expander is present
+          if (
+            __SERVER__ &&
+            !hasApiExpander('types', getBaseUrl(location.pathname))
+          ) {
+            return dispatch(getTypes(getBaseUrl(location.pathname)));
+          }
+        },
+      },
+      {
+        key: 'workflow',
+        promise: ({ location, store: { dispatch } }) =>
+          __SERVER__ && dispatch(getWorkflow(getBaseUrl(location.pathname))),
+      },
+    ]),
+    injectIntl,
+    connect(
+      (state, props) => ({
+        pathname: props.location.pathname,
+        token: state.userSession.token,
+        userId: state.userSession.token
+          ? jwtDecode(state.userSession.token).sub
+          : '',
+        content: state.content.data,
+        apiError: state.apierror.error,
+        connectionRefused: state.apierror.connectionRefused,
+      }),
+      null,
+    ),
+  )(AppComponent);
+}
+
+export default connectAppComponent(App);
```

Comment:
So default export is 'connected' then named export is bare component with bare name, right?

Trying to establish conventions (and forward thinking in `@plone/components` as well).

How about naming `connectedAppComponent` instead?